### PR TITLE
Firelike drawtype: Improve code

### PR DESCRIPTION
**Pull request by paramat**
Remove unusable fine rotation by param2
Remove unused and redundant code
Fix code style issues

Design and behaviour of firelike is unchanged.

### CODE REVIEW

**Comment by est31** [comment]
Looks good. The fine rotation wasn't even documented, and I guess less likely used, so I'm OK with a remove.

File: src/content_mapblock.cpp [hunk 1]
```diff
@@ -1165,148 +1165,123 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
...
-				{
-					for(int i = 0; i < 4; i++) {
-						vertices[i].Pos.rotateXZBy(90 + n.param2 * 2);
+				if (j == 0 && (drawAllFaces ||
+						(doDraw[3] == 1 || doDraw[1] == 1))) {
+					for (int i = 0; i < 4; i++) {
+						vertices[i].Pos.rotateXZBy(90);
 						vertices[i].Pos.rotateXYBy(-10);
-						vertices[i].Pos.Y -= vOffset;
+						//vertices[i].Pos.Y -= vOffset;
```
Comment (est31) [inline src/content_mapblock.cpp]: why did you comment this out, and not removed completely? (happens at 3 other places as well).

**Comment by paramat** [comment]
I could remove it, thought it might be useful if in future the side textures were tilted differently, but that's not likely so i'll remove those lines.

**Comment by paramat** [comment]
8e9c9e305aecdb5b432c0990fef85fa6ff3c042b

**Pull request closed without merging**